src/components/PasswordToggle/PasswordToggle.test.tsx

```diff
@@ -0,0 +1,36 @@
+import { mount, shallow } from "enzyme";
+import React from "react";
+
+import PasswordToggle from "./PasswordToggle";
+
+describe("PasswordToggle ", () => {
+  it("renders", () => {
```

Comment:
I'm -0 on these types of tests. As _any_ change to make to the component will make the tests fail. If the component doesn't render, you'll know in the first real test below.

Reply:
We add them, because we see them in other components so it seems it is a default part of the test suit.
Personally I also wonder about usefulness of them, because I've never seen them fail to reveal any issue. They fail when you change component and you expect it to change.

Seems like we should have a team-wide decision if and when to use snapshot tests. And if we don't want them, remove them. Because right now the seem like standard in react-component repo.

Reply:
I suspect the mostly exist from when these components existed in MAAS. Since then we've largely moved away from snapshot testing in MAAS.